engine/file_storage.py

```diff
@@ -14,7 +14,7 @@ class FileStorage():
     def save(save_game):
         '''FileStorage module to create a saved game'''
 #        save_game.__achievements.append(achievement.check_achievements())
-        with open('save_game.json', 'w') as file:
+        with open('save_game.json', 'a+') as file:
```

Comment:
We should probably write instead of appending, unless overwriting save files isn't allowed. If we append, the dict might also get confused when saving. My thought is, read the file, update the dict with the new save, then save to the file using the w flag

Reply:
That might be inefficient though

Reply:
But then again, especially if we're appending, that'll really confuse the save since it'll be multiple dictionaries which'll break the json, or a list of dictionaries which could confuse the reload